sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/LoadBalancingRule.java

```diff
@@ -33,6 +34,13 @@ public interface LoadBalancingRule
     /** @return the probe associated with the load balancing rule */
     LoadBalancerProbe probe();
 
+    /**
+     * Gets the list of backends associated with the load balancing rule.
+     *
+     * @return the list of backends associated with the load balancing rule
+     */
+    List<LoadBalancerBackend> backends();
```

Comment:
Do we need to add some comments in original `backend()`, e.g. `returns the first load balancing rule if multiple exists`?

Reply:
Sure, will do.

I assume we don't need `@deprecate` at present stage, as backend has yet to remove `backendAddressPool` (but from my test, they don't have value in it).